src/main/scala/com/redis/cluster/RedisClusterOps.scala

```diff
@@ -47,6 +53,22 @@ trait RedisClusterOps extends AutoCloseable {
     nodeForKey(key).withClient(body(_))
   }
 
+  // todo: need Cats Semigroup instance for T to combine, add Cats?
+  // https://typelevel.org/cats/typeclasses/semigroup.html
```

Comment:
One of the targets in this library was to keep it free of any dependencies. We could add `cats` but then I would design the library in a different way using `cats-effects` to model all effects ..

Reply:
Sure. It is not a big deal at the moment but since most of the api is based on Option[A] it's difficult to model cluster api with Iterable[A] result to fit it.

To be honest I don't know what is best now :) I can delete the comment, however it will probably be a topic for further discussion at some point anyway

Reply:
i just left todo with a note, i hope its fine.